### PR TITLE
cnf-tests: look for suitable node for SRIOV/NUMA

### PR DESCRIPTION
**Pull request by zeeke**
If the first node in the list returned by
`sriovcluster.DiscoverSriov(...)` doesn't have devices
on multiple NUMA nodes, check if the other nodes can
be used for the test suite.

Fixes
- https://github.com/openshift-kni/cnf-features-deploy/issues/1688

cc @evgenLevin, @gregkopels

### CODE REVIEW

**Review by evgenLevin — COMMENTED** [review]
Looks good. Just 1 comment

File: cnf-tests/testsuites/e2esuite/dpdk/numa_node_sriov.go [hunk 1]
```diff
@@ -63,20 +63,13 @@ var _ = Describe("[sriov] NUMA node alignment", Ordered, func() {
...
-		Expect(len(sriovCapableNodes.Nodes)).To(BeNumerically(">", 0))
-		testingNode, err := nodes.GetByName(sriovCapableNodes.Nodes[0])
-		Expect(err).ToNot(HaveOccurred())
-		By("Using node " + testingNode.Name)
 
-		sriovDevices, err := sriovCapableNodes.FindSriovDevices(testingNode.Name)
-		Expect(err).ToNot(HaveOccurred())
+		var testingNode *corev1.Node
+		testingNode, numa0DeviceList, numa1DeviceList = findTestingNodeWithDevicesInTwoNUMANodes(sriovCapableNodes)
+		Expect(len(sriovCapableNodes.Nodes)).To(BeNumerically(">", 0))
```
[the remark this answers is not in the document]
Comment (zeeke) [reply cnf-tests/testsuites/e2esuite/dpdk/numa_node_sriov.go:69]: Good catch!

**Comment by evgenLevin** [comment]
lgtm

**Comment by zeeke** [comment]
/override ci/prow/e2e-gcp-ovn

know failing test `[It] [rfe_id:27363][performance] CPU Management when pod runs with the CPU load balancing runtime class [test_id:32646] should disable CPU load balancing for CPU's used by the pod `

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-kni_cnf-features-deploy/1689/pull-ci-openshift-kni-cnf-features-deploy-master-e2e-gcp-ovn/1727719092879626240

**Pull request merged**